.github/workflows/lint.yml

```
@@ -1,45 +1,69 @@
           python -m venv ~/.venv/ci-venv
+
+      - name: Install dependencies and build
+        run: |
           source ~/.venv/ci-venv/bin/activate
           python -m pip install --upgrade pip
           pip install -r python/requirements-dev.txt
           make develop
-      - name: Linting with ruff
+
+      - name: Linting with Ruff
         run: |
```

actionlint

shellcheck reported issue in this script: SC1090:warning:1:8: ShellCheck can't follow non-constant source. Use a directive to specify location

(shellcheck)